### PR TITLE
Update python sample snippets to regenerate samples for CRM

### PR DESCRIPTION
**Pull request by hzyi-google**

### CODE REVIEW

File: toolkit/src/test/java/com/google/api/codegen/testdata/discoveries/py/py_adexchangebuyer.v1.4.json.baseline [hunk 1]
```diff
@@ -410,11 +410,11 @@ credentials = None
 
 service = discovery.build('adexchangebuyer', 'v1.4', credentials=credentials)
 
-request = service.creatives().list()
-while request is not None:
+while True:
```
Comment (vchudnov-g) [inline toolkit/src/test/java/com/google/api/codegen/testdata/discoveries/py/py_adexchangebuyer.v1.4.json.baseline]: how do we break out of this loop? Same for many of the other ones in this PR. Am I missing something obvious?

**Comment by hzyi-google** [comment]
Now it should work... @vchudnov-g PTAL

**Comment by hzyi-google** [comment]
@vchudnov-g Another look please?

**Comment by JustinBeckwith** [comment]
👋 @vchudnov-g we're trying out a new PR SLO, and this one is out :) Mind taking a look?

File: toolkit/src/test/java/com/google/api/codegen/testdata/discoveries/py/py_cloudresourcemanager.v1.json.baseline [hunk 2]
```diff
@@ -130,8 +130,7 @@ search_organizations_request_body = {
 }
 
 while True:
-    request = service.organizations().search(body=search_organizations_request_body)
-    response = request.execute()
+request = service.organizations().search(body=search_organizations_request_body)    response = request.execute()
```
Comment (vchudnov-g) [inline toolkit/src/test/java/com/google/api/codegen/testdata/discoveries/py/py_cloudresourcemanager.v1.json.baseline]: indentation

File: toolkit/src/main/resources/com/google/api/codegen/py/sample.snip [hunk 3]
```diff
@@ -99,8 +99,14 @@
 
     @end
     @if class.isPageStreaming
-        while True:
+        @if class.pageStreaming.isResourceSetterInRequestBody
+            while True:
```
Comment (vchudnov-g) [inline toolkit/src/main/resources/com/google/api/codegen/py/sample.snip]: the following line needs to be indent (see baseline)

**Pull request merged**